### PR TITLE
add configuration for maximum sessions & subscriptions

### PR DESCRIPTION
**Pull request by cristipogacean**
## Purpose
add CLI options to control the:
* maximum session count
* maximum subscription count

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

```
```

## What to Check
Verify that the following are valid
* ...

### CODE REVIEW

**Review by koepalex — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -337,6 +337,12 @@ Options:
       --to, --trustowncert   the own certificate is put into the trusted
                                certificate store automatically.
                                Default: False
+      --msec, --maxsessioncount
+                             maximum number of paralel sessions.
```
Comment (luiscantero) [inline README.md:341]: typo
```suggestion
                             maximum number of parallel sessions.

```

File: src/CliOptions.cs [hunk 2]
```diff
@@ -86,6 +86,9 @@ public static Mono.Options.OptionSet InitCommandLineOptions()
 
             { "to|trustowncert", $"the own certificate is put into the trusted certificate store automatically.\nDefault: {TrustMyself}", (string s) => TrustMyself = s != null },
 
+            { "msec|maxsessioncount=", $"maximum number of paralel sessions.\nDefault: {MaxSessionCount}", (ushort i) => MaxSessionCount = i },
```
Comment (luiscantero) [inline src/CliOptions.cs:89]:
```suggestion
            { "msec|maxsessioncount=", $"maximum number of parallel sessions.\nDefault: {MaxSessionCount}", (ushort i) => MaxSessionCount = i },

```

File: src/CliOptions.cs [hunk 3]
```diff
@@ -86,6 +86,9 @@ public static Mono.Options.OptionSet InitCommandLineOptions()
 
             { "to|trustowncert", $"the own certificate is put into the trusted certificate store automatically.\nDefault: {TrustMyself}", (string s) => TrustMyself = s != null },
 
+            { "msec|maxsessioncount=", $"maximum number of paralel sessions.\nDefault: {MaxSessionCount}", (ushort i) => MaxSessionCount = i },
+            { "msuc|maxsubscriptioncount=", $"maximum number of subscriptions.\nDefault: {MaxSubscriptionCount}", (ushort i) => MaxSubscriptionCount = i },
```
Comment (luiscantero) [inline src/CliOptions.cs:90]: Question: We use ushort for the param here, but the SDK expects an int. Does setting it to a negative number have a meaning that might be useful?

File: src/CliOptions.cs (hunk 2 above)
Comment (dacolgit) [inline src/CliOptions.cs:89]: It should be with double l => parallel

File: README.md [hunk 4]
```diff
@@ -337,6 +337,12 @@ Options:
       --to, --trustowncert   the own certificate is put into the trusted
                                certificate store automatically.
                                Default: False
+      --msec, --maxsessioncount
```
Comment (luiscantero) [inline README.md:340]: Pls update version.json to 2.9.4

**Review by dacolgit — CHANGES_REQUESTED** [review]

File: src/OpcApplicationConfiguration.cs [hunk 5]
```diff
@@ -208,6 +210,11 @@ public async Task<ApplicationConfiguration> ConfigureAsync()
 
         // show certificate store information
         await ShowCertificateStoreInformationAsync().ConfigureAwait(false);
+        ApplicationConfiguration.ServerConfiguration.MaxSessionCount = MaxSessionCount;
+        ApplicationConfiguration.ServerConfiguration.MaxSubscriptionCount = MaxSubscriptionCount;
+        Logger.Information("Application configured with MaxSessionCount: {maxSessionCount} and MaxSubscriptionCount: {maxSubscriptionCount}",
```
Comment (mregen) [inline src/OpcApplicationConfiguration.cs]: add these setting in the serverBuilder with the fluent API

**Review by karok2m — APPROVED** [review]

**Review by luiscantero — APPROVED** [review]
:shipit:

File: src/OpcApplicationConfiguration.cs (hunk 5 above)
Comment (cristipogacean) [reply src/OpcApplicationConfiguration.cs]: sure!

**Review by karok2m — APPROVED** [review]

**Pull request merged**